hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -185,6 +185,11 @@
               --img-cfg {gen_otp_images_cfg_dir}/otp_ctrl_img_raw.hjson \
               --out {run_dir}/otp_ctrl_img_raw.vmem \
               {gen_otp_images_cmd_opts}
+        ''',
+        '''{gen_otp_images_cmd} \
+              --img-cfg {gen_otp_images_cfg_dir}/otp_ctrl_img_test_unlocked0.hjson \
+              --out {run_dir}/otp_ctrl_img_test_unlocked0.vmem \
+              {gen_otp_images_cmd_opts}
```

Comment:
We probably have to align this with #15319, depending on which PR lands first.

Reply:
Done.